libs/shared/db/mysql/account/project.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "shared-db-mysql-account",
+  "$schema": "../../../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/shared/db/mysql/account/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nx/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/libs/shared/db/mysql/account",
+        "tsConfig": "libs/shared/db/mysql/account/tsconfig.lib.json",
+        "packageJson": "libs/shared/db/mysql/account/package.json",
+        "main": "libs/shared/db/mysql/account/src/index.ts",
+        "assets": ["libs/shared/db/mysql/account/*.md"]
+      }
+    },
```

Comment:
I believe we want a compile target here as well.

Reply:
The generator doesn't seem to include one. The paypal one has no compile target either.